docs/source/Htypes.rst

```diff
@@ -99,6 +99,12 @@ Video Htype
 
 - :bluebold:`Sample dimensions:` ``(# frames, height, width, # channels)`` :bluebold:`or` ``(# frames, height, width)``
 
+:blue:`Limitations`
+-------------------
+- Visualization of videos in the Deep Lake App is limited by the video coded support for various browsers. 
+    - `Chrome supports the codecs shown here <https://www.chromium.org/audio-video/>`_.
+- :ref:`The Deep Lake Performant Dataloader <Dataloader>` does not support videos. Please use the `OSS Dataloader <Dataloader>`
```

Comment:
Hey @FayazRahman How do I make the dataloader references point to the right place. I can't seem to figure it out.

Reply:
When using `:ref:`, the name in `<name>` points to an anchor somewhere else in the docs defined by `.. _name:`
So wherever you want the `:ref:` to point to, we should add `.. _dataloader:` just above it.